plugins/compositor/compositorwindow.cpp

```diff
@@ -82,19 +82,19 @@ void CompositorWindow::initialize(QWaylandXdgSurface *shellSurface)
 void CompositorWindow::forceVisible()
 {
     QtWayland::ExtendedSurface *pExtendedSurfaceExt = static_cast<QtWayland::ExtendedSurface*>(surface()->extension(QtWayland::ExtendedSurface::interfaceName()));
-    pExtendedSurfaceExt->sendOnScreenVisibilityChange(true);
+    if(pExtendedSurfaceExt) pExtendedSurfaceExt->sendOnScreenVisibilityChange(true);
```

Comment:
Easier to read when you would put it inside of {} ? Same applies for the others.